addons/web_editor/static/src/scss/wysiwyg_snippets.scss

```diff
@@ -819,8 +819,6 @@ body.editor_enable.editor_has_snippets {
         // Buttons
         we-button.o_we_user_value_widget {
             > div {
-                // Needed otherwise cannot work because of flex display
-                @include o-text-overflow(block);
```

Comment:
Kinda scare by the fact you remove a line which is explained as "Needed otherwise..." without explanation and that your commit message says "Needs more testing" :grimacing: 

With this line removed, you lose the "..." effect in button whose text is too long so that can't be the right solution here